docs/useCases.md

```diff
@@ -478,15 +478,16 @@ publishDataset.execute(datasetId, versionUpdateType)
 _See [use case](../src/datasets/domain/useCases/PublishDataset.ts) implementation_.
 
 The above example publishes the dataset with the specified identifier and performs a minor version update. If the response
-is successful, the use case does not return the dataset object, but the HTTP status code `200`. Otherwise, it throws an error.
-If you want to perform a major version update, you must set the `versionUpdateType` parameter to `VersionUpdateType.MAJOR`.
-
+is successful, the use case does not return the dataset object, but the HTTP status code `200`. Otherwise, it throws an error.\
+If you want to perform a major version update, you must set the `versionUpdateType` parameter to `VersionUpdateType.MAJOR`.\
+Superusers can pass `VersionUpdateType.UPDATE_CURRENT` to update metadata without changing the version number. This will overwrite the latest published version and therefore will only work for a dataset that has been published at least once. \*Note that this will only work also if there were no file changes in the update.\
 The `datasetId` parameter can be a string, for persistent identifiers, or a number, for numeric identifiers.
```

Comment:
I know we haven't worked on permissions yet generally, but it could be good to add a comment that UPDATE_CURRENT is only available to 'superuser' accounts, since it's more of a special case than other permissions logic.  But if we want to wait till later, that's ok.

Reply:
Oh, I'm sorry, I misread that comment in the doc!  Yes, that looks perfect. :)